test/elm/convert/data.cql

```diff
@@ -122,3 +122,53 @@ define IsNull: ToBoolean('falsetto')
 // @Test: ToConcept
 define IsValid: ToConcept(Code { system: 'http://loinc.org', code: '8480-6' }) // Concept { codes: { Code { system: 'http://loinc.org', code: '8480-6' } } }
 define IsNull: ToConcept(null as Code)
+
+// @Test: ConvertsToBoolean
+define IsTrueWithTrueValue: ConvertsToBoolean('y')
+define IsTrueWithFalseValue: ConvertsToBoolean('0')
+define IsFalse: ConvertsToBoolean('foo')
+define IsNull: ConvertsToBoolean(null as String)
+
+// @Test: ConvertsToDate
+define IsTrue: ConvertsToDate('2015-01-01')
+define IsFalse: ConvertsToDate('foo')
+define IsNull: ConvertsToDate(null as Date)
+
+// @Test: ConvertsToDateTime
```

Comment:
There is another signature for `ConvertsToDateTime` that sort of throws a wrench in things:
`ConvertsToDateTime(argument Date) Boolean`

Reply:
I am missing something, what's the issue?

Reply:
Looks like ToDateTime properly handles Dates, but it would be good to add a test for it.  The failure I was seeing was related to a different issue.